plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -253,6 +257,7 @@ public void cancel() throws InterruptedException {
             getId());
     info("log file path is: " + logFilePath);
 
-    HadoopJobUtils.proxyUserKillAllSpawnedHadoopJobs(logFilePath, jobProps, tokenFile, getLog());
```

Comment:
reverted most of the format changes the remaining ones are the fix for the existing code which doesn't apply for the azkaban format